services/graph-protocol/docker-compose.yaml

```diff
@@ -15,16 +17,19 @@ services:
       postgres_user: graph-node
       postgres_pass: let-me-in
       postgres_db: graph-node
-      ipfs: 'http://host.docker.internal:5001'
+      ipfs: 'https://api.thegraph.com/ipfs/'
```

Comment:
Is this the part where Docker IPFS isn't working properly on Linux?

Reply:
yes